server/chat-commands/info.ts

```diff
@@ -211,7 +211,7 @@ export const commands: ChatCommands = {
 			}
 
 			if (targetUser.semilocked) {
-				buf += `<br />Semilocked: ${targetUser.semilocked}`;
+				buf += `<br />Semilocked: ${user.can('lock') ? targetUser.semilocked : "(reason hidden)"}`;
```

Comment:
```suggestion
				buf += `<br />Semilocked: ${user.trusted ? targetUser.semilocked : "(reason hidden)"}`;
```

I think I prefer a lower threshold like `trusted`. Is that fine with you?